x-pack/plugins/security_solution/public/assistant/overlay.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { AssistantOverlay as ElasticAssistantOverlay } from '@kbn/elastic-assistant';
+import { useAssistantContext } from '@kbn/elastic-assistant/impl/assistant_context';
```

Comment:
nit: Consider exporting `useAssistantContext` via the packages's public API in `x-pack/packages/kbn-elastic-assistant/index.ts` to avoid importing directly from the package's `impl` directory